package.json

```diff
@@ -23,6 +23,7 @@
     "mocha": "latest",
     "plist": "latest",
     "typescript": "latest",
+    "vscode-grammar-updater": "^1.0.4",
```

Comment:
Why do we need this..

Reply:
Ah, meant to revert that. I was trying to simplify testing the tmLanguage in VS Code since it looks like they switched to pre-generating a .JSON file from the tmLanguage for TypeScript so I couldn't just copy/paste.